macros/secondary_calculations/secondary_calculation_period_over_period.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro default__secondary_calculation_period_over_period(metric_name, grain, dimensions, calc_config) -%}
+{%- macro default__secondary_calculation_period_over_period(metric_name, grain, dimensions, calc_config, metric_config_dict) -%}
```

Comment:
Same comment about not including the datatype in the metric (or, I guess I could get on board with renaming `calc_config` to `calc_config_dict`, if you think clarifying the datatype is important? I just like consistency)